packages/backend/src/models/RolesModel.ts

```diff
@@ -462,42 +474,63 @@ export class RolesModel {
-            createdAt: Date;
-        }>
-    > {
-        const userAssignments = await this.database('organization_memberships')
-            .join('users', 'organization_memberships.user_id', 'users.user_id')
-            .join(
-                'organizations',
-                'organization_memberships.organization_id',
-                'organizations.organization_id',
-            )
-            .leftJoin(
-                'roles',
-                'organization_memberships.role_uuid',
-                'roles.role_uuid',
-            )
-            .select(
-                'roles.role_uuid as roleId',
-                'roles.name as roleName',
-                'users.user_uuid as assigneeId',
-                this.database.raw(
-                    "CONCAT(users.first_name, ' ', users.last_name) as assigneeName",
-                ),
-                'organizations.organization_uuid as organizationId',
-                'organization_memberships.created_at as createdAt',
-            )
-            .where('organizations.organization_uuid', orgUuid)
-            .whereNotNull('organization_memberships.role_uuid');
+    // eslint-disable-next-line class-methods-use-this
+    private mapOrganizationRoleAssignment(
+        userAssignments: DbOrganizationRoleAssignment[],
+    ): RoleAssignment[] {
+        const formattedUserAssignments: RoleAssignment[] = userAssignments.map(
+            (assignment) => ({
+                roleId: assignment.customRoleUuid || assignment.roleName,
+                roleName: assignment.customRoleName || assignment.roleName,
+                assigneeType: 'user' as const,
+                ownerType: assignment.ownerType || 'system',
+                assigneeId: assignment.assigneeId,
+                assigneeName: assignment.assigneeName,
+                organizationId: assignment.organizationId,
+                createdAt: assignment.createdAt,
+                updatedAt: assignment.createdAt, // Use createdAt since updatedAt doesn't exist
+            }),
+        );
+        return formattedUserAssignments;
+    }
 
-        return userAssignments;
+    async getOrganizationRoleAssignments(
+        orgUuid: string,
+    ): Promise<RoleAssignment[]> {
+        const userAssignments: DbOrganizationRoleAssignment[] =
+            await this.database('organization_memberships')
+                .join(
+                    'users',
+                    'organization_memberships.user_id',
+                    'users.user_id',
+                )
+                .join(
+                    'organizations',
+                    'organization_memberships.organization_id',
+                    'organizations.organization_id',
+                )
+                .leftJoin(
+                    'roles',
+                    'organization_memberships.role_uuid',
+                    'roles.role_uuid',
+                )
+                .select(
+                    `${RolesTableName}.role_uuid as customRoleUuid`,
```

Comment:
Do we prefer interpolating the table name like this or aliasing the table in the join? I've seen both used. How do we decide whether to use the constant or the raw string?